### ISSUE TITLE
[Picviewer CE+]How can I disable it on specific websites?

### ISSUE THREAD

**Issue by AiraNadih** [hoothin/UserScripts#425]
**Describe the bug**
I used the following settings, but it didn't work:
```
[
    {
    name: "Xiangqi.com",
    url: /https:\/\/play\.xiangqi\.com\/.*/,
    enabled: false
    }
]
```

**To Reproduce**
1. Go to 'https://play.xiangqi.com/'

**Expected behavior**
If I understand correctly, if you set `enabled: false` , the floating button should no longer be displayed.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome, firefox
 - Version 115, 115

**Comment by hoothin** [comment]
No, the term 'enabled' simply indicates that the current rule will not have any impact. You should use 'exclude' inside scripts manager.
